game/scripts/npc/abilities/shadow_demon_shadow_poison.txt

```diff
@@ -54,7 +54,7 @@
       "03"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "bonus_stack_damage"                              "50"
+        "bonus_stack_damage"                              "50 50 50 50 100 200"
```

Comment:
Just to scale into OAA levels. Who gives a fuck about 50 damage at level 40 somethin lol

Reply:
I think the reason this doesn't scale is because it used to scale and it was too much damage? Because it's damage per stack? Anyways don't forget about the talent that increases this damage by a percentage.

Reply:
There is 0 chance this spell will ever scale to OAA levels. You could buff the shit out of it to try and artificially make it relevant but its probably better just to leave it.

Reply:
with the talent at max oaa level it's 230 damage so it's fine imho.